### DISCUSSION TITLE
Problema dispositivi bticino / legrand integrati tramite homekit dopo aggiornamento HASSIO a 2022.2.0

### DISCUSSION THREAD

**Discussion by Alexanderdavice** [indomus/forum#1448] [HUB: Home Assistant]
Buongiorno a tutti,
dopo l'aggiornamento di HASSIO alla versione  2022.2 sto riscontrando alcuni problemi con i dispositivi Bticino Living Now integrati in Home Assistant tramite HomeKit.
Nello specifico tutte le entità sono indisponibili, mentre i device sembrano ancora collegati correttamente. Non rilevo però messaggi specifici da log e registro.

Ho notato che dopo l'aggiornamento si sono generate in automatico delle nuove entitià tipo button (es. button.rolling_shutter_switch_1_identify per le tapparelle, button.micromodule_switch_1_identify per il relè smart, button.smart_power_outlet_1_identify per le prese connesse e button.switch_1_identify per i deviatori connessi) e che, nella pagina dei dispositivi, è comparsa la scheda "Diagnostica" con comando "Identify" e "PREMI" associato alle nuove entità button (immagini allegate).

Potrebbe essere che l'introduzione degli Input Button della nuova versione siano la causa del malfunzionamento?
Eventualmente come posso investigare meglio il problema?

Ho effettuato anche l'aggiornamento alla 2022.2.1 senza notare miglioramenti.

Aggiornamento 07/02/2022: eseguito aggiornamento a ultima release 2022.2.3, anche in questo caso senza successo. I dispositivi sporadicamente sono disponibili e funzionanti, anzi sembra che siano disponibili per qualche secondo per poi tornare indisponibili (allego foto dello stato). Ho anche notato che nell'ultimo periodo sono presenti alcuni malfunzionamenti sull'app Bticino, per cui potrebbe anche essere un problema lato cloud Bticino (verificherò anche questo)

grazie mille in anticipo! 
Ale

![1](https://user-images.githubusercontent.com/62716606/152518795-278adf98-da04-4c8e-a4d1-976ff4a2ec97.png)
![2](https://user-images.githubusercontent.com/62716606/152518797-7bbd52b2-f135-4995-9751-34b315f82665.png)
![stato luce cameretta - Home Assistant](https://user-images.githubusercontent.com/62716606/152803710-831b4f3f-5ec9-4a51-a354-7bd8dd0b6079.png)

**Comment by essegi65** [comment]
Io sono alla versione di HA 2022.2.2 e li uso (diversi deviatori connessi e prese comandate) con l'integrazione homekit controller, sia con l'app bticino. però non ho questi problemi ne con HA ne con app bticino. in effetti da un aggiornamento recente mi sembra quello di novembre o dicembre mi sono apparsi nell'integrazione homekit controller anche dei dispositivi dimmerswitch che però non capisco a cosa servono e non hanno entità configurabile

**Comment by Alexanderdavice** [comment]
Verificando con assistenza Bticino mi è stato consigliato di effettuare il reset del gateway. Ho comunque insistito con la riconfigurazione del wifi e dopo molteplici tentativi di ri-associazione del gateway sono riuscito a collegarlo e ad effettuare l'aggiornamento firmware.
Da quel momento sembra che il sistema abbia ripreso a funzionare normalmente sia su app Home+control che su HA.
In ogni caso ne ho approfittato per associare le prese connesse a conbee (su deconz) in modo da avere il dettaglio dei consumi disponibile in Home Assitant (cosa non possibile con integrazione mediante homekit). Aspetto qualche giorno e verifico che effettivamente non ci siano più malfunzionamenti per poi poter chiudere il thread.
Per intanto grazie

**Reply by genga91** [reply]
Ciao a tutti, anche io sto avendo problemi con luci e prese living now, sinceramente non so da che versione perché non uso frequentemente luci e prese da HA. Avete sviluppi? Avete fatto reset del gateway quindi?
Mi ricordate come si fa? Mi ricordo che qualche mese fa feci il reset ma dovetti riassociare tutte le luci, un disastro... quindi prima di muovermi attendo un vostro feedback. Grazie :)

**Reply by Alexanderdavice** [reply]
Ciao,
la prima operazione da eseguire è quella di togliere tensione all'impianto e riattivarla dopo una decina di secondi. In qualche occasione per me ha funzionato.
E' possibile una procedura di reset "soft" del gateway che corrisponde alla ri-configurazione della rete del gateway stesso è che non fa perdere la configurazione con HA. Si esegue premendo il tastino del gateway (attenzione del gateway, non del comando entra ed esci associato): ma con una pressione breve si resetta la rete, mentre con una pressione prolungata si fa un reset completo (non ricordo però i tempi precisi di pressione tasto).
Anche nel manuale del gateway non è specificato ([Link](https://www.catalogo.bticino.it/app/webroot/low_res/485871_493514_LE10964AA.pdf)
Spero di esserti stato d'aiuto, io comunque ho ri-segnalato la cosa all'assistenza bticino

**Reply by genga91** [reply]
Ciao, ho risolto ieri sera, confermo che un riavvio del gateway risolve il problema, basta togliere e dare di nuovo alimentazione al gateway è tutto torna disponibile. Grazie !

**Reply by Alexanderdavice** [reply]
@genga91 : a te il problema è capitato solo una volta?

Io ho visto che il problema sembra ripetersi, quindi non posso continuamente togliere e dare di nuovo alimentazione al gateway..

Sono appena stato contattato dall'assistenza
Hanno verificato che il segnale del router è molto buono e mi hanno consigliato di:
-effettuare il reset rete del gateweay (pressione pulsante reset per 10 s, il led arancio lampeggia)
-settare il gateway con ip fisso, abilitando poi la porta 25050 TCP

Se anche così i problemi persistono hanno consigliato di cambiare il gateway.
Vi aggiorno..

**Reply by icricelli** [reply]
Seguo

**Reply by bordeo** [reply]
Capita la stessa cosa anche a me. Il gateway si disconnette completamente dal router. Solo dopo giorni non so per quale motivo puo' succedere che si riconnette. A volte mi rimane connesso per molti giorni. Non sono ancora riuscito ad individuare la causa e lo utilizzo da circa 2 anni.
Una soluzione a cui stavo pensavo è di mettere uno shelly/sonoff 1canale attaccato al gateway per staccare la corrette e riattaccarla e magari agganciare un'automazione per farlo quando il gateway non è più raggiungibile.

**Reply by Alexanderdavice** [reply]
È  esattamente quello che ho fatto con un sonoff a monte del gateway che effettua un riavvio e invia notifica ( me ne arrivano anche 2 o 3 al giorno). Soluzione facile ed economica per non dover sostituire il gateway con tutto quello che comporta..nell'attesa di aver voglia di passare tutto direttamente sotto HA..

**Reply by icricelli** [reply]
Ho anche io questo problema da 3 mesi, ho provato tutte le soluzioni suggerite in rete (impostare ip fisso al gateway, aprire la porta 25050 TCP, associare un singolo access point al gateway stesso, disattivare il band steering, allontanare qualsiasi device Wi.fi presente nel raggio di 3 metri). Nessun risultato. Il prossimo passo è cambiare fornitore di Access Point per escludere sia un problema di non compatibilità con UNIFI, voi che marche usate?

**Reply by bordeo** [reply]
Io ho usato per un anno un [Technicolor TIM](https://www.tim.it/assistenza/assistenza-tecnica/guide-manuali/technicolor) e poi un altro anno un d-link. Anch'io davo la colpa alla rete o alle impostazioni di homeassistant ma a sentire dai vostri racconti secondo me è proprio un problema del gateway BTicino e l'unica maniera per ovviare è riavviarlo quando si disconnette.

**Comment by gurreri** [comment]
Ciao a tutti
provo a sottoporre a voi un problema magari sapete darmi un feedback

ho comprato qualche giorno da un gateway matix bticino con comando entra/esci wireless.
ho associato prodotto in primis con sua app bticino e poi in home assistant per integrarli
ma in home assistant vedo solo il gateway non il comando wireless. secondo voi come mai? in questo modo non posso creare scenari utilizzando il comando entra/esci

il gateway lo vedo come integrazione dell'app homekit e non sotto la voce netatmo (essendo integrazione che integra anche prodotti legrand e bticino). ho provato ad aggiornare e ricaricare ma niente il gateway lo vedo con integrazione homekit...ma null'altro

grazie per qualsiasi feedback.  ciao

**Reply by mirkoboggian** [reply]
Ciao, probabilmente arrivo tardi. 

Anche io avevo un sacco di problemi di disconnessione continua e ho risolto tutto quando fastweb mi ha attivato un "IP v4 Statico Pubblico" (partendo da un IPv6 dinamico privato) . Da quel momento non ho più avuto problemi relativi alla connessione.

**Reply by nolith** [reply]
ciao @gurreri anche io ho lo stesso problema con il master entra/esci wireless che si vede solo nell'app bticino ma non in HA.

Tu hai risolto?

**Comment by Argebombo** [comment]
Ciao a tutti, ho avuto lo stesso problema...

In più c'ho messo del mio; ho pensato anche io a qualche problema di rete ma ho comunque deciso di provare a riconfigurare il tutto; Guardando sul libretto di istruzioni ho resettato la connessione wifi e homekit per ripristinare l'integrazione, prima però (e qui arriva il mio problema) ho eliminato la configurazione non funzionante da HA.

Sono riuscito a resettare il gateway, solo che adesso, quando cerco di attivare l'integrazione "dispositivo Homekit" dice che "Non è stato possibile trovare dispositivi non associati". Quindi anche se ho resettato il gateway, per HA è rimasta una sorta di associazione al dispositivo.

Sapete come posso fare per eliminare la "memoria" della vecchia configurazione?

Sono su Core vers. 2023.10.3


Grazie e Buona Giornata a tutti.

**Reply by nolith** [reply]
Allora io ho avuto un problema simile, avevo associato il gateway direttamente a homekit del cellulare e HA non lo voleva più associare perché lo vedeva già preso. Anche scollegando il gateway da homekit o facendo il reset rimaneva sempre già "collegato" per HA.

Alla fine sono riuscito a ri-associarlo forzando la procedura dall'app home + controll andando in gestione casa, selezionando il gateway, e premendo configura homekit. In questo modo il gateway si dichiara libero e appare nuovamente in HA, però l'ho dovuto fare tipo 3-4 volte perché spesso si abbinava prima al telefono che ad HA e risultava impossibile da collegare.

**Comment by Argebombo** [comment]
> Allora io ho avuto un problema simile, avevo associato il gateway direttamente a homekit del cellulare e HA non lo voleva più associare perché lo vedeva già preso. Anche scollegando il gateway da homekit o facendo il reset rimaneva sempre già "collegato" per HA.
> 
> Alla fine sono riuscito a ri-associarlo forzando la procedura dall'app home + controll andando in gestione casa, selezionando il gateway, e premendo configura homekit. In questo modo il gateway si dichiara libero e appare nuovamente in HA, però l'ho dovuto fare tipo 3-4 volte perché spesso si abbinava prima al telefono che ad HA e risultava impossibile da collegare.

Grazie per la Risposta!! Purtroppo io non ho quell'impostazione, questo è quello che vedo:

![Screenshot](https://github.com/indomus/forum/assets/62376000/a0fe3f3c-5a0b-4132-87d8-542e6667d1fa)

Comunque sono riuscito tramite reset delle impostazioni Wifi+Homekit come spiegato qui:

![immagine](https://github.com/indomus/forum/assets/62376000/1566d3f4-056c-41a1-abac-4517112535b9)

E ci sono riuscito perchè ho dovuto riconfigurare dall'app il wifi.

Il problema è che ho eliminato l'integrazione non funzionante da HA, ma sembra che il gateway, sempre per HA, sia ancora accoppiato.

Qualche Idea?

**Reply by mirkoboggian** [reply]
Ciao, stesso problema. Tutto funzionava poi dopo un aggiornamento la parte HomeKit ha smesso di funzionare.. se per caso hai un dispositivo apple, per far apparire la scritta “imposta homekit”, devi prima aprire l’app Casa e disassociare i dispositivi. Ho fatto una prova con un termostato ed effettivamente appare in HA anche se non sono riuscito a collegarlo prima che ci riuscisse l’app BTicino-Casa. Domani ci riprovo..se nel frattempo riesci a collegare nuovamente il GW fammi sapere..

**Reply by mirkoboggian** [reply]
Ciao, sono riuscito a ri-aggiungere un termostato. La procedura fatta (non è detto che sia l'ottimale) su iPhone è stata:

1. Reset del Termostato
2. Eliminazione da progetto Home + Control
3. Riassociazione del temostato in Home + Control
4. Disssociazione del termostato da "CASA"

In questo modo è rimasta l'associazione in Home + Control ed è apparso il "dispositivo HomeKit" in HA.
Se riesco entro sera provo con un altro termostato. Più complessa penso sarà la procedura del Gateway.

**Reply by mirkoboggian** [reply]
Confermo, la procedura funziona anche col gateway.

**Comment by gurreri** [comment]
in realtà non ho risolto per davveo ma ho trovato alcuni codici per
individuare la pressione dei tasti e creando cosicun scena

ti devo spiegare meglio con screen. appena ho due min ti dico

Il Gio 19 Ott 2023, 19:14 Alessio Caiazza ***@***.***> ha
scritto:

**Reply by nolith** [reply]
Scusa mi era sfuggita la risposta. 

Attendo gli screenshot perché sono molto interessato

**Reply by acastellacci** [reply]
> in realtà non ho risolto per davveo ma ho trovato alcuni codici per individuare la pressione dei tasti e creando cosicun scena ti devo spiegare meglio con screen. appena ho due min ti dico Il Gio 19 Ott 2023, 19:14 Alessio Caiazza ***@***.***> ha scritto:
> […](#)
> ciao @gurreri <https://github.com/gurreri> anche io ho lo stesso problema con il master entra/esci wireless che si vede solo nell'app bticino ma non in HA. Tu hai risolto? — Reply to this email directly, view it on GitHub <[#1448 (reply in thread)](https://github.com/indomus/forum/discussions/1448#discussioncomment-7330749)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ATP344CUUC2DPLNIZLH3TH3YAFNYHAVCNFSM5NRPM7C2U5DIOJSWCZC7NNSXTOKENFZWG5LTONUW63SDN5WW2ZLOOQ5TOMZTGA3TIOI> . You are receiving this because you were mentioned.Message ID: ***@***.***>

Ciao,
Ho lo stesso problema, puoi chiamarmi e condividere come hai risolto?

**Reply by Vinz87** [reply]
> in realtà non ho risolto per davveo ma ho trovato alcuni codici per
> 
> individuare la pressione dei tasti e creando cosicun scena
> 
> 
> 
> ti devo spiegare meglio con screen. appena ho due min ti dico
> 
> 
> 
> Il Gio 19 Ott 2023, 19:14 Alessio Caiazza ***@***.***> ha
> 
> scritto:
> 
> 
> 
> > ciao @gurreri <https://github.com/gurreri> anche io ho lo stesso problema
> 
> > con il master entra/esci wireless che si vede solo nell'app bticino ma non
> 
> > in HA.
> 
> >
> 
> > Tu hai risolto?
> 
> >
> 
> > —
> 
> > Reply to this email directly, view it on GitHub
> 
> > <https://github.com/indomus/forum/discussions/1448#discussioncomment-7330749>,
> 
> > or unsubscribe
> 
> > <https://github.com/notifications/unsubscribe-auth/ATP344CUUC2DPLNIZLH3TH3YAFNYHAVCNFSM5NRPM7C2U5DIOJSWCZC7NNSXTOKENFZWG5LTONUW63SDN5WW2ZLOOQ5TOMZTGA3TIOI>
> 
> > .
> 
> > You are receiving this because you were mentioned.Message ID:
> 
> > ***@***.***>
> 
> >
> 
> 

Ciao puoi condividere questi codici?

**Comment by Alexanderdavice** [comment]
Anche io molto interessato..

Il Ven 4 Set 2026, 07:31 Vinz ***@***.***> ha scritto:
